mod/05_email/set/type/email_template.rb

```diff
@@ -44,7 +44,7 @@ def email_config args={}
   
   [:subject, :text_message].each do |field_name|
     config[field_name] = process_email_field( field_name, args ) do |field_card|
-      field_card.contextual_content context_card, {:format=>'email_text'}, args
+      field_card.contextual_content context_card, {:format=>'email_text'}, args.merge(:content_opts=>{ :chunk_list=>:inclusion_and_link})
```

Comment:
I was imprecise. wagn.org became wagn.org\[http://wagn.org] ( not wagn.org[wagn.org] ). That's probably fine and how it is supposed to be for text format.

For emails I suggest that we process the following chunk types:
* subject: only inclusions, no url, no link,
* text body: inclusions and links (with default text format), no url
* html body: all chunks
* address: inclusions, error for url,  special format for links  [[href | text]] -> text\<href>

specs are coming soon